apps/aecore/test/aec_chain_tests.erl

```diff
@@ -269,15 +255,14 @@ get_work_test_() ->
                BH0 = aec_blocks:to_header(B0),
                ?assertEqual({ok, BH0}, aec_chain:top_header()),
 
-               %% Check difficulty of genesis - for readability of the test.
-               1.0 = aec_headers:difficulty(BH0),
+               %% Check difficulty of genesis
+	       %% NOTE: floating point matching... not safe.
```

Comment:
My guess is it's *pretty* safe in this case, but still, if it's regarded as unsafe, wouldn't it be better to change it? E.g. into something like `trunc(1000 * ?GENESIS_DIFFICULTY) = trunc(1000, ...)`